### PR TITLE
Adding copyright/license string on top of all ".py" files

### PR DESCRIPTION
**Pull request by dkaranjai**
Signed-off-by: pulirupu geetha <puliruprx.geetha@intel.com>
Signed-off-by: pulirupu geetha <pulirupux.geetha@intel.com>

### CODE REVIEW

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-51-1/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/f4582ef677f05816c245bf91f5b57b107497e3ff

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-51-2/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/f4582ef677f05816c245bf91f5b57b107497e3ff

**Review by 5abeel — APPROVED** [review]
lgtm

**Review by nupuruttarwar — APPROVED** [review]
Looks good. Please rebase the branch before merging

**Comment by dkaranjai** [comment]
Please let us know if this can be merged once the jenkins checks have passed.

**Comment by ipdksw** [comment]
Result:ABORTED JenkinsConsole:https://s3.ipdk.io/ovs/PR-51-6/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/54cfa8eb7961fb01c5676cc73d0ea82d464b84df

File: ptf_tests/common/lib/__init__.py
Comment (5abeel) [inline ptf_tests/common/lib/__init__.py:2]: Please add the following to after the Copyright line in all files

`SPDX-License-Identifier: Apache-2.0`

See example in this PR: https://github.com/ipdk-io/ovs/pull/52/files

**Review by 5abeel — CHANGES_REQUESTED** [review]

File: ptf_tests/common/lib/__init__.py
Comment (dkaranjai) [reply ptf_tests/common/lib/__init__.py:2]: Sure.

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-51-10/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6b6f5af89038b116ca7bf6a64ae985ea75b22809

**Comment by ipdksw** [comment]
Result:ABORTED JenkinsConsole:https://s3.ipdk.io/ovs/PR-51-11/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6b6f5af89038b116ca7bf6a64ae985ea75b22809

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-51-12/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/6b6f5af89038b116ca7bf6a64ae985ea75b22809

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-51-13/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/819d747428341c01567cff00f76e1f7720325386

File: ptf_tests/common/utils/ovsp4ctl_utils.py
Comment (5abeel) [inline ptf_tests/common/utils/ovsp4ctl_utils.py:2]: Missing license info. Please add

SPDX-License-Identifier: Apache-2.0

File: ptf_tests/tests/DPDK_Hot_Plug_Remove.py
Comment (5abeel) [inline ptf_tests/tests/DPDK_Hot_Plug_Remove.py:2]: Missing license info. Please add:

SPDX-License-Identifier: Apache-2.0

File: ptf_tests/tests/DPDK_Hot_Plug_Remove_Multi_port.py
Comment (5abeel) [inline ptf_tests/tests/DPDK_Hot_Plug_Remove_Multi_port.py:2]: Missing

SPDX-License-Identifier: Apache-2.0

File: ptf_tests/common/lib/__init__.py
Comment (5abeel) [inline ptf_tests/common/lib/__init__.py:8]: Lots of empty lines in all the files in this commit. This can be cleaned up to reduce the license block

**Review by 5abeel — CHANGES_REQUESTED** [review]

**Comment by ipdksw** [comment]
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-51-14/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/97da8d59f61533210169c1c0b96c644f37986c68

**Comment by dkaranjai** [comment]
@5abeel have addressed the comments

**Review by 5abeel — APPROVED** [review]
lgtm

**Pull request merged**
